python/whylogs/core/schema.py

```diff
@@ -102,6 +105,7 @@ def resolve(self, *, pandas: Optional[pd.DataFrame] = None, row: Optional[Mappin
 
                 self._columns[k] = ColumnSchema(
                     dtype=type(v),
+                    cfg=self.default_configs,
```

Comment:
pass the frozen MetricConfig to the ColumnSchema so that config will apply to each column.